signing_service/src/tests/tests.py

```diff
@@ -0,0 +1,59 @@
+from unittest import TestCase
+import mock
+import sys
+
+from ..constants import SIGNING_SERVICE_DEFAULT_PORT
+from ..signing_service import _parse_arguments
+from ..signing_service import main
+
+
+class SigningServiceMainTestCase(TestCase):
+
+    def test_that_main_should_called_with_correct_address(self):
```

Comment:
I would suggest sth like `test_that_wrong_ip_address_causes_system_exit`

Reply:
It can be a host name also.